### PR TITLE
Enable feature vote

### LINKED ISSUE #1173: Voting on feature requests
We need to display current star priority at a minimum

### PR DESCRIPTION
**Pull request by nanaya**
With very basic interface :dancer: 

Now with stars :stars: 

Resolves #1173.

### CODE REVIEW

**Comment by nanaya** [comment]
adding stars

**Comment by nanaya** [comment]
added stars

**Comment by nekodex** [comment]
You may want to double-check the queries...

File: resources/views/forum/topics/_feature_vote.blade.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            {!! trans('forum.topics.show.feature_vote.info._', [
+                'feature_request' => '<strong>'.trans('forum.topics.show.feature_vote.info.feature_request').'</strong>',
+                'supporters' => link_to(route('support-the-game'), trans('forum.topics.show.feature_vote.info.supporters')),
+            ]) !!}
+        </div>
+
+        <div
+            class="
+                forum-topic-feature-vote__votes
+                {{ $topic->osu_starpriority > 1000 ? 'forum-topic-feature-vote__votes--small' : '' }}
```
Comment (nekodex) [inline resources/views/forum/topics/_feature_vote.blade.php]: might be worth moving this to a config variable?

File: resources/lang/en/sort.php [hunk 2]
```diff
@@ -16,25 +18,15 @@
...
+    'new' => 'Recent',
+    'old' => 'Old',
+    'top' => 'Top',
 
-  &__item-arrow {
-    margin-left: 5px;
-  }
-}
+    'forum_topics' => [
+        'new' => 'Last replied',
```
Comment (nekodex) [inline resources/lang/en/sort.php]: "Last replied" kinda implies _my_ last reply, so maybe "Last reply"? (also might want to update the topic listing to match too)

File: resources/lang/en/forum.php [hunk 3]
```diff
@@ -268,6 +269,12 @@
...
                 'do' => 'Promote this request',
 
+                'info' => [
+                    '_' => 'This is a :feature_request. Feature requests can be voted up by :supporters.',
+                    'feature_request' => 'feature request',
+                    'supporters' => 'supporters',
+                ],
+
                 'user' => [
                     'count' => '{0} no vote|{1} :count vote|[2,*] :count votes',
```
Comment (nekodex) [inline resources/lang/en/forum.php]: Noticed this while testing... not due to PR changes but probably should still be fixed -- `no vote` should be `no votes`...



(may need to doublecheck where else the string is used)

**Pull request merged**